eva_sub_cli/validators/validator.py

```diff
@@ -445,6 +457,52 @@ def _convert_metadata_attribute(self, sheet, json_attribute, xls2json_conf):
             if attributes_dict[attribute] == json_attribute:
                 return attribute
 
+    def _collect_md5sum_to_metadata(self):
+        md5sum_file = resolve_single_file_path(os.path.join(self.output_dir, 'md5sums.txt'))
+        file_path_2_md5 = {}
+        file_name_2_md5 = {}
+        if md5sum_file:
+            with open(md5sum_file) as open_file:
+                for line in open_file:
+                    sp_line = line.split()
+                    md5sum = sp_line[0]
+                    vcf_file = sp_line[1]  # assuming no space in the vcf_file_name
+                    file_path_2_md5[vcf_file] = md5sum
+                    file_name_2_md5[os.path.basename(vcf_file)] = md5sum
+        if self.metadata_json_post_validation:
+            with open(self.metadata_json_post_validation) as open_file:
+                try:
+                    json_data = json.load(open_file)
+                    analysis_aliases = [a.get('analysisAlias') for a in json_data.get('analysis', [])]
+                    file_rows = []
+                    if len(analysis_aliases) == 1:
```

Comment:
Is the separate logic for a single analysis really necessary?

Reply:
That was to cover the case when no files are provided but that was not implemented correctly.
I now have changed it.
It's not necessary but without it there will be more failures.

Reply:
Won't the no files provided case be covered by the [metadata validation](https://github.com/EBIvariation/eva-sub-cli/blob/main/eva_sub_cli/etc/eva_schema.json#L14)? So it should be caught earlier in the process.

Reply:
I've removed it now.